src/commands/development/paginate.js

```diff
@@ -0,0 +1,96 @@
+import { Command } from 'discord.js-commando'
+import { RichEmbed } from 'discord.js'
+import { respondWithPaginatedEmbed } from '../../utils/embed'
+
+let items = [
+  { name: 'evan', value: 'so cool they even have their own emoji' },
+  { name: 'gusto', value: 'benevolent dictator for life <3' },
+  { name: 'elfayer', value: 'unrivaled at archery (except by Robin Hood)' },
+  { name: 'naito', value: 'has degree in TypeScript witchcraft' },
+  { name: 'lloyd', value: 'is practically neighbours with sustained' },
+  { name: 'akryum', value: 'is there anything he cannot do' },
+  { name: 'dobromir', value: '#most-attractive-moderator' },
+  { name: 'dr. taco', value: 'mexican food specialist and meme expert' },
+  { name: 'joe', value: 'has gifs for all possible occasions' },
+  { name: 'dave stewart', value: 'saviour of all vuex users' },
+  { name: 'galvez', value: 'best hat in all the lands' },
+]
+items = [...items, ...items, ...items]
+
+/*
+  This development-only command allows one to test the pagination utility.
+*/
+module.exports = class DevelopmentPaginateCommand extends Command {
+  constructor(client) {
+    super(client, {
+      name: 'paginate',
+      args: [
+        {
+          key: 'itemsPerPage',
+          type: 'integer',
+          prompt: 'how many items to display per page?',
+          default: 2,
+          validate(value) {
+            return value <= items.length
+          },
```

Comment:
Is it fine to validate in the constructor while most data would be loaded asynchronously?

Reply:
I see what you're saying - usually you wouldn't be able to validate like this, that's right and it would need to happen in `run` but maybe it's okay for the test command?